### PR TITLE
Count the number of Set bits of an integer

### PR DESCRIPTION
**Pull request by Sameera-Aryasomayajula**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/other/countsetbit.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+int main()
+{
+    int n;
+    cin>>n;
+    cout<< countbits(n)<<endl;
+    
+    return 0;
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/other/countsetbit.cpp:28]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make apt README changes and I will merge it

**Comment by atarax665** [comment]
@Sameera-Aryasomayajula Are you still working on this? Please reply to reopen this PR.

**Comment by Sameera-Aryasomayajula** [comment]
I am working on this and will make the requested changes.

**Comment by HarshCasper** [comment]
Closing the PR cause of no updates. Let me know if you would like this to be reopened.

**Pull request closed without merging**